charts/policy-controller/templates/webhook/cleanup-leases.yaml

```diff
@@ -25,6 +25,12 @@ spec:
             - -c
             - kubectl delete leases --all --ignore-not-found -n {{ .Release.Namespace }}
       restartPolicy: OnFailure
+      {{- if .Values.leasescleanup.securityContext.enabled }}
```

Comment:
you might dont need the enabled variable, just to check if the object exists should be fine

Reply:
I was trying to [conform to the style used elsewhere in the helm chart](https://github.com/sigstore/helm-charts/blob/main/charts/policy-controller/templates/webhook/deployment_webhook.yaml#L111-L116), I'm happy to change it however if consistency isn't as important

Reply:
ok, sgtm to keep